lib/matplotlib/mpl-data/stylelib/cheatsheet_gallery.mplstyle

```diff
@@ -0,0 +1,28 @@
+# from the Matplotlib cheatsheet as used in our gallery:
+
+
+axes.facecolor: white
+axes.edgecolor: black
+axes.linewidth: 1.0
+axes.grid: True
+axes.axisbelow: True       # grid/ticks are below elements (e.g., lines, text)
+
+xtick.color: 555555
```

Comment:
why do you need to set x/ytick rcs?
(also, quite a few other rcs seem to just match the default value?)

Reply:
Tried to minimize it.

Reply:
Can't we have fully transparent color for the ticks ? On the gallery, I can see part of them (on the left [here](https://55035-1385122-gh.circle-artifacts.com/0/doc/build/html/plot_types/B_arrays/d_quiver.html#sphx-glr-plot-types-b-arrays-d-quiver-py) for example.

Reply:
Sure, I've made their length 0